package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "scratch-desktop",
-  "productName": "Scratch Desktop",
+  "productName": "Scratch",
```

Comment:
The NPM `productName` is used inside the app for things like the title bar, "About" window, etc. -- basically any time `productName` is mentioned in a JavaScript file under `src/`